joint_limits:
  gripper_jaw_1:
    has_position_limits: true
    has_velocity_limits: true
    max_velocity: 1
    has_acceleration_limits: true
    max_acceleration: 1
    angle_wraparound: false
  gripper_jaw_2:
    has_position_limits: true
    has_velocity_limits: true
    max_velocity: 1
    has_acceleration_limits: true
    max_acceleration: 1
    angle_wraparound: false
  joint1:
    has_position_limits: true
    has_velocity_limits: true
    max_velocity: 4.36
    has_acceleration_limits: true
    max_acceleration: 1
    angle_wraparound: false
  joint2:
    has_position_limits: true
    has_velocity_limits: true
    max_velocity: 4.36
    has_acceleration_limits: true
    max_acceleration: 1
    angle_wraparound: false
  joint3:
    has_position_limits: true
    has_velocity_limits: true
    max_velocity: 4.36
    has_acceleration_limits: true
    max_acceleration: 1
    angle_wraparound: false
  joint4:
    has_position_limits: true
    has_velocity_limits: true
    max_velocity: 5.58
    has_acceleration_limits: true
    max_acceleration: 1
    angle_wraparound: false
  joint5:
    has_position_limits: true
    has_velocity_limits: true
    max_velocity: 5.58
    has_acceleration_limits: true
    max_acceleration: 1
    angle_wraparound: false
  joint6:
    has_position_limits: true
    has_velocity_limits: true
    max_velocity: 7.33
    has_acceleration_limits: true
    max_acceleration: 1
    angle_wraparound: false